core/component.go

```diff
@@ -129,6 +130,7 @@ const (
 type Component struct {
 	Identity         *pb_discovery.Announcement
 	Discovery        pb_discovery.Client
+	NOC              pb_noc.MonitorClient
```

Comment:
@johanstokking: That is not true, as it is an abbreviation of Network Operations Center. Similar to Internet Protocol being `IP` and with `ID`, `EUI` etc. Unfortunately the generated protos are different, but this is not a Go convention.